addons/mail/models/mail_activity_plan.py

```diff
@@ -70,3 +71,9 @@ def _compute_has_user_on_demand(self):
         self.has_user_on_demand = False
         for plan in self.filtered('template_ids'):
             plan.has_user_on_demand = any(template.responsible_type == 'on_demand' for template in plan.template_ids)
+
+    @api.returns('self', lambda value: value.id)
+    def copy(self, default=None):
+        default = default or {}
+        default['name'] = self.name + _(' (copy)')
```

Comment:
We should not override name, it case it's in default
```suggestion
        if 'name' not in default:
            default['name'] = self.name + _(' (copy)')
```